Packs/ApiModules/Scripts/JSONFeedApiModule/JSONFeedApiModule_test.py

```diff
@@ -350,3 +357,109 @@ def test_fetch_indicators_command_google_ip_ranges(mocker):
     indicators, _ = fetch_indicators_command(client, indicator_type=None, feedTags=[], auto_detect=None, limit=100)
     for indicator in indicators:
         assert indicator.get('value')
+
+
+def test_json_feed_with_config_mapping_with_aws_feed_no_update(mocker):
+    """
+    Given
+    - Feed config from AWS feed, with last_run from the same feed, emulating the first
+      fetch after updating the AWS Feed integration when there is no update to the feed.
+
+    When
+    - Running fetch indicators command
+
+    Then
+    - Ensure that the correct message displays in demisto.debug, and the last_run object
+     remained the same, and continue to have the previous AWS feed config name 'AMAZON'.
+    """
+    with open('test_data/amazon_ip_ranges.json') as ip_ranges_json:
+        ip_ranges = json.load(ip_ranges_json)
+
+    mocker.patch.object(demisto, 'debug')
+    last_run = mocker.patch.object(demisto, 'setLastRun')
+
+    feed_name_to_config = {
+        'AMAZON$$CIDR': {
```

Comment:
shouldn't it be:
```suggestion
        'AMAZON': {
```

Reply:
let's add assertions on this

Reply:
No, because the config name itself changed to "AMAZON$$CIDR", in the mock_last_run we have the previous config name "AMAZON"